src/tests/FunctionalTests/WebAssembly/Browser/RuntimeConfig/WebAssembly.Browser.RuntimeConfig.Test.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TestRuntime>true</TestRuntime>
+    <Scenario>WasmTestOnBrowser</Scenario>
+    <ExpectedExitCode>42</ExpectedExitCode>
+    <WasmMainJSPath>runtime.js</WasmMainJSPath>
+    <EnableDefaultItems>false</EnableDefaultItems>
+  </PropertyGroup>
+  
+  <ItemGroup>
+    <Compile Include="Program.cs" />
+    <Content Include="index.html">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
+
+  <Target Name="AfterWasmBuildApp" AfterTargets="WasmBuildApp">
+    <Copy SourceFiles="$(OutDir)\index.html" DestinationFolder="$(WasmAppDir)" />
+  </Target>
```

Comment:
The `<Content..` can be dropped, since we are explicitly copying it in the `AfterWasmBuildApp` target anyway. And in the `Copy`, we can use `<Copy SourceFiles="$(MSBuildThisFileDirectory)index.html" DestinationFolder="$(WasmAppDir)" />`

Reply:
This is indeed necessary. Otherwise, I hit this error
```
/Users/fanyang/Documents/work/dotnet_wasm/runtime/src/tests/FunctionalTests/WebAssembly/Browser/RuntimeConfig/WebAssembly.Browser.RuntimeConfig.Test.csproj(14,5): error MSB3030: Could not copy the file "/Users/fanyang/Documents/work/dotnet_wasm/runtime/artifacts/bin/WebAssembly.Browser.RuntimeConfig.Test/net6.0-Release/browser-wasm/index.html" because it was not found.
```

Reply:
Yep, that's why I suggested:
`<Copy SourceFiles="$(MSBuildThisFileDirectory)index.html" DestinationFolder="$(WasmAppDir)" />`

note that it uses `$(MSBuildThisFileDirectory)index.html`.

What happens here is that the `@(Content)` item will cause the file to be copied to the `$(OutDir)`. And that would end up in `publish` dir, but `WasmAppBuilder` overwrites that with it's own `index.html`. So, `AfterTargets="WasmBuildApp"`, we explicitly want to overwrite with our `index.html`.

And we don't run from `$(OutDir)`, so we don't really need `index.html` there. So, instead of going through all of that, we can directly copy `$(MSBuildThisFileDirectory)index.html` to the app bundle dir.

Reply:
Ah gotcha!